### ISSUE TITLE
TZUNTIL missing UTC modifier

### ISSUE THREAD

**Issue by bobvandevijver** [iCalcreator/iCalcreator#104]
The latest releases (v2.41.56) seems to have dropped the UTC modifier from the TZUNTIL field, which I believe is in error when looking at the RFC: https://www.rfc-editor.org/rfc/rfc7808.html#section-5.3.4

```
1) App\Tests\Service\User\UserIcalServiceTest::testCustomEventCalendar
Failed asserting that two strings are equal.
--- Expected
+++ Actual
@@ @@
 X-LIC-LOCATION:Europe/Amsterdam\n
 BEGIN:VTIMEZONE\n
 TZID:Europe/Amsterdam\n
-TZUNTIL:20231029T010000Z\n
+TZUNTIL:20231029T010000\n
 BEGIN:STANDARD\n
 TZNAME:CET\n
 DTSTART:20211031T030000\n
```

**Comment by iCalcreator** [comment]
Fixed in 2.41.61, please test.

**Comment by bobvandevijver** [comment]
Looks fixed, thanks 👍🏻
